radontea/_version.py

```diff
@@ -57,7 +57,7 @@
     import traceback
     import warnings
 
-    def git_describe():
+    def git_describe() -> str:
```

Comment:
Please revert your changes to _version.py, it is a static/stable file shared across repositories and should not be modified at this time. Thanks!